packages/sit-onyx/src/components/OnyxLink/OnyxLink.vue

```diff
@@ -0,0 +1,43 @@
+<script lang="ts" setup>
+import type { OnyxLinkProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxLinkProps>(), {
```

Comment:
didn't we want to set href to empty string as default?
![image](https://github.com/SchwarzIT/onyx/assets/151019977/802e7eae-4eae-4cf4-a0d8-7daffcf26f88)

e.g. in https://github.com/SchwarzIT/onyx/issues/294 the link is only used as click handler without navigating thus the href does not have to be set

Reply:
Ah, it's conform with the ticket AC of #237 . guess we'll then need to set the empty href when implementing #294 on that instance.

Reply:
question goes to @mj-hof